hack/build-manifests.sh

```
@@ -243,6 +243,19 @@ create_autopilot_csv() {
 
+function create_vm_file_restore_operator_csv() {
+  local operatorName="vm-file-restore-operator"
+  local dumpCRDsArg="--dump-crds"
+  local operatorArgs=" \
+    --csv-version=${CSV_VERSION} \
+    --namespace=${OPERATOR_NAMESPACE} \
+    --operator-version=${VM_FILE_RESTORE_OPERATOR_VERSION} \
+    --operator-image=${VM_FILE_RESTORE_OPERATOR_IMAGE} \
+    --pull-policy=IfNotPresent \
+  "
+  gen_csv ${DEFAULT_CSV_GENERATOR} ${operatorName} "${VM_FILE_RESTORE_OPERATOR_IMAGE}" ${dumpCRDsArg} ${operatorArgs}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)